components/popover/demo/basic.md

```diff
@@ -1,10 +1,18 @@
 ---
 order: 0
-title: 基本
+title: 
+  zh-CN:基本
+  en-US:Basic
 ---
 
+## zh-CN
+
 最简单的用法，浮层的大小由内容区域决定。
 
+## en-US
+
+The most basic way. The size of the floating layer depends on the region of contents.
```

Comment:
The most basic example. The size of the floating layer depends on the region of contents.